Kentor.AuthServices/SAML2P/Saml2Response.cs

```diff
@@ -369,6 +369,7 @@ private void ValidateSignature(IOptions options)
         /// <summary>Checks the signature.</summary>
         /// <param name="signedRootElement">The signed root element.</param>
         /// <param name="idpKey">The assymetric key of the algorithm.</param>
+        [System.Diagnostics.CodeAnalysis.SuppressMessage( "Microsoft.Naming", "CA2204:Literals should be spelled correctly", MessageId = "AuthServices" )]
```

Comment:
Better to add AuthServices to CustomDictionary.xml (I'm surprised it isn't already listed there).